contracts/Cross-chain/OmnichainGovernanceExecutor.sol

```diff
@@ -220,6 +225,12 @@ contract OmnichainGovernanceExecutor is ReentrancyGuard, BaseOmnichainController
             bytes[] memory calldatas,
             uint8 pType
         ) = abi.decode(payload, (address[], uint256[], string[], bytes[], uint8));
+        require(
+            targets.length == values.length &&
+                targets.length == signatures.length &&
+                targets.length == calldatas.length,
+            "OmnichainGovernanceExecutor::_nonblockingLzReceive: proposal function information arity mismatch"
```

Comment:
Related to CRO-06: `arity`?

Reply:
Arity is refers to: The number of arguments that a function can take. We have also used this term in Governance.
https://github.com/VenusProtocol/governance-contracts/blob/feat/ven-1918/contracts/Governance/GovernorBravoDelegate.sol#L200